File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    
    bool check(vector<int>& nums) {
       int c = 0;
       for(int i = 1; i < nums.size(); i++){
            if(nums[i - 1] > nums[i]) c+=1;
       }

       if(nums[nums.size()-1] > nums[0]) c+=1;

       return c<=1;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        ans = 0
        for i in range(1 , len(nums)):
            if(nums[i - 1] > nums[i]):
                ans += 1
        
        
        if nums[-1] > nums[0]:
            ans += 1

        return ans <= 1
        